styles.css

```diff
@@ -36,26 +41,55 @@ button {
   font-size: 1.1em;
 }
 
-button:hover {
+.normal-btn:hover {
   box-shadow: 1px 8px 16px #36b37f3d;
 }
 
-button:active {
+.normal-btn:active {
   background: #008552;
 }
 
 /* Head of website */
 
 .title {
-  color: #227150;
+  color: #2d9368;
 }
 
 .menu {
   display: none;
+  font-weight: 600;
+  font-size: 32px;
```

Comment:
- [ ] Your nav menu works fine, however, the design is not convincing me, check the image below and take care of fixing what is shown there, 
<img width="1165" alt="1" src="https://github.com/Arch-Noize/portfolio-microverse/assets/77843050/eae4dbed-921f-4833-b762-6fc9b4f34763">